example/ios/Podfile.lock

```diff
@@ -653,4 +653,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: adf35afb016478dc51ce14a884b7366d6ecee4c0
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.0
```

Comment:
@samarthjain10 revert this change, update your local cocoapods to the latest one by running `sudo gem install cocoapods`

Reply:
Then run `pod install` in the ./example/ios folder